common/lib/xmodule/xmodule/video_module/transcripts_utils.py

```diff
@@ -640,25 +670,39 @@ def available_translations(self, transcripts, verify_assets=None):
 
         # If we've gotten this far, we're going to verify that the transcripts
         # being referenced are actually in the contentstore.
+        val_transcript_languages = []
+        if verify_val_transcripts:
+            val_transcript_languages = get_available_transcript_languages(
+                edx_video_id=self.edx_video_id,
+                youtube_id_1_0=self.youtube_id_1_0,
+                html5_sources=self.html5_sources
+            )
+
         if sub:  # check if sjson exists for 'en'.
             try:
                 Transcript.asset(self.location, sub, 'en')
             except NotFoundError:
                 try:
                     Transcript.asset(self.location, None, None, sub)
                 except NotFoundError:
-                    pass
+                    # Check in edx-val transcript languages
+                    if 'en' in val_transcript_languages:
+                        translations.append('en')
                 else:
-                    translations += ['en']
+                    translations.append('en')
             else:
-                translations += ['en']
+                translations.append('en')
```

Comment:
This `if...else` and `try...except` blocks if hard to understand. Can we simplify it?